src/python/pants/reporting/zipkin_reporter.py

```diff
@@ -115,3 +118,10 @@ def end_workunit(self, workunit):
     if workunit in self._workunits_to_spans:
       span = self._workunits_to_spans.pop(workunit)
       span.stop()
+
+  def close(self):
+    """End the report."""
+    endpoint = self.endpoint.replace("/api/v1/spans", "")
+
+    logger.info("Zipkin trace ID {}".format(self.trace_id))
+    logger.info("Zipkin trace may be located at this URL {}/traces/{}".format(endpoint, self.trace_id))
```

Comment:
Let's only log this line, not the one above, and let's maybe log at `debug` rather than `info` level, as this is information most people won't want most of the time, but may want to see when they're specifically trying to debug :)

Reply:
^This is what we do with buildstats internally (print a log at `debug` level), and we only print a warning if we can't connect to the buildstats server to upload.

Reply:
IMO: in v1, it should be shown all the time as long as the run isn't `--quiet`.

Rather than the buildstats model (or maybe in-addition-to?), another model that might be useful would be the `./pants server` model. If the pants server is running, you see:
![see-a-report-at](https://user-images.githubusercontent.com/46740/53842906-ab497980-3f55-11e9-92f2-f3756947d41f.png)

Reply:
So, concretely: yes... maybe this particular message should be at debug level.

But finding where the `See a report at ...` message and logging there as well would be useful.

Reply:
I'm going to merge for now; I'm not sure concretely what the follow-up you're asking for here is (is it unconditionally logging in a second place?), but we can treat it as a follow-up :)

Reply:
> (is it unconditionally logging in a second place?)
Yes. See the "See a report at:" message in the screenshot.